hrjobcontract/js/src/job-contract/controllers/modal/modal-contract-new.js

```diff
@@ -49,6 +49,14 @@ define([
                 is_primary: 0
             };
 
+            // Since we are adding a new Contract, we set the values for each leave type with the
+            // AbsenceTypes values
+            $scope.entity.leave.forEach(function(leave, index) {
+              var absenceTypeID = $scope.entity.leave[index].leave_type;
```

Comment:
Let the code breathe a little, leave a new line between `var` declaration/definition and the rest of the code
```js
var absenceTypeID = $scope.entity.leave[index].leave_type;

$scope.entity.leave[index].leave_amount = utils.absenceTypes[absenceTypeID].default_entitlement;
$scope.entity.leave[index].add_public_holidays = utils.absenceTypes[absenceTypeID].add_public_holiday_to_entitlement;
```